### PR TITLE
Add listener response

### PR DESCRIPTION
**Pull request by yyyogev**

### CODE REVIEW

File: logziosender.go [hunk 1]
```diff
@@ -207,21 +205,20 @@ func (l *LogzioSender) Stop() {
...
 	defer resp.Body.Close()
 	statusCode := resp.StatusCode
-
-	_, err = io.Copy(ioutil.Discard, io.LimitReader(resp.Body, respReadLimit))
+	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		l.debugLog("Error reading response body: %v", err)
 	}
-	return statusCode
+	return statusCode, string(body)
```
Comment (idohalevi) [inline logziosender.go]: We can print the warning/error message here instead of returning another value

File: logziosender.go [hunk 2]
```diff
@@ -243,7 +240,7 @@ func (l *LogzioSender) shouldRetry(attempt int, statusCode int) bool {
 	}
 
 	if !retry && statusCode != http.StatusOK {
-		l.requeue()
+		l.debugLog("got error response from server: " + responseMsg)
```
Comment (idohalevi) [inline logziosender.go]: I think we can take it off if we change the code above according to my comment

File: logziosender.go [hunk 3]
```diff
@@ -145,7 +143,7 @@ func SetDrainDuration(duration time.Duration) SenderOptionFunc {
 	}
 }
 
-// SetDrainDiskThreshold to change the maximum used disk space
+// SetCheckDiskSpace to check if it crosses the maximum allowed disk usage
```
Comment (idohalevi) [inline logziosender.go:146]: Thanks! :)

File: test/test.go [hunk 4]
```diff
@@ -19,7 +19,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/logzio/logzio-go"
+	"logzio-go"
```
Comment (idohalevi) [inline test/test.go:22]: Thanks again

**Review by idohalevi — CHANGES_REQUESTED** [review]
minor changes

**Comment by yyyogev** [comment]
@idohalevi done changes

**Review by idohalevi — APPROVED** [review]
PERFECT!

**Pull request merged**